src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxInvoicePluginApi.java

```diff
@@ -78,20 +78,30 @@ public AvaTaxInvoicePluginApi(final AvaTaxConfigurationHandler avaTaxConfigurati
     }
 
     @Override
-    public List<InvoiceItem> getAdditionalInvoiceItems(final Invoice invoice, final boolean dryRun, final Iterable<PluginProperty> properties, final CallContext context) {
+    public AdditionalItemsResult getAdditionalInvoiceItems(final Invoice invoice, final boolean dryRun, final Iterable<PluginProperty> properties, final InvoiceContext context) {
         final Collection<PluginProperty> pluginProperties = Lists.<PluginProperty>newArrayList(properties);
 
         final Account account = getAccount(invoice.getAccountId(), context);
 
         checkForTaxExemption(invoice, pluginProperties, context);
         checkForTaxCodes(invoice, pluginProperties, context);
 
-        try {
-            return calculator.compute(account, invoice, dryRun, pluginProperties, context);
-        } catch (final Exception e) {
-            // Prevent invoice generation
-            throw new RuntimeException(e);
-        }
+        return new AdditionalItemsResult() {
```

Comment:
Ideally, we should use the automatically generated POJOs for this. We might need to regenerate them first (see https://github.com/killbill/killbill-plugin-framework-java/issues/82).